### DISCUSSION TITLE
New Feature Ideas

### DISCUSSION THREAD

**Discussion by JacksonMan9000** [CubeLabsNZ/CubeTime#51] [Ideas]
I think it would be a great idea to add in different subsets of scrambles such as the ones you would find on cstimer. It would be great to have 2 gen scrambles so that we could practice f2l and look ahead. Also, it would be cool if there were a scramble type that set up pll or oll so you could practice those times.

**Comment by xbjfk** [comment]
We a rough idea for a complete alg trainer, however subsets before then might be a good idea. (continued in #54)
